compassion_dashboard/README.rst

```diff
@@ -0,0 +1,27 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
No I mean is the whole module `compassion_dashboard` necessary? The module `partner_communication` is not even depending on it.